### PR TITLE
vstte paper

### PR DESCRIPTION
**Pull request by rahulku**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 and MIT licenses.

### CODE REVIEW

**Review by carolynzech — APPROVED** [review]

**Comment by carolynzech** [comment]
This isn't going to merge automatically because it fails our required copyright status check. I believe we'll have to temporarily disable this check (or at least make it optional) so that this can merge.

**Comment by tautschnig** [comment]
> @rahulku This isn't going to merge automatically because it fails our required copyright status check. I believe we'll have to temporarily disable this check (or at least make it optional) so that this can merge.

Perhaps we should just include adequate copyright information in each of the files?

**Comment by carolynzech** [comment]
> > @rahulku This isn't going to merge automatically because it fails our required copyright status check. I believe we'll have to temporarily disable this check (or at least make it optional) so that this can merge.
> 
> Perhaps we should just include adequate copyright information in each of the files?
I figured it wasn't appropriate to put our copyright on someone else's work?

File: papers/vstte2024/llncs.cls [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
```
Comment (tautschnig) [inline papers/vstte2024/llncs.cls:3]: We must not do that. That file has a copyright already.

**Review by tautschnig — CHANGES_REQUESTED** [review]
I suggest we have a `Makefile` or shell script that downloads the LNCS template files from Springer's website and extracts the necessary files.

**Comment by tautschnig** [comment]
> I suggest we have a `Makefile` or shell script that downloads the LNCS template files from Springer's website and extracts the necessary files.

The following should do the trick:
````sh
#!/bin/sh
curl https://resource-cms.springernature.com/springer-cms/rest/v1/content/19238648/data/v8 -o llncs.zip
unzip llncs.zip llncs.cls splncs04.bst
rm llncs.zip
````

File: papers/vstte2024/llncs.cls (hunk 1 above)
Comment (celinval) [reply papers/vstte2024/llncs.cls:3]: We could however add a modifications copyright. We would still need to add `%` as a [valid comment starter here](https://github.com/model-checking/kani/blob/6ac5183eabd704c04b88f80552bedf24f7c8e4e9/scripts/ci/copyright_check.py#L11-L19).

If we don't want to add a copyright statement, than @rahulku would need to add it to the [copyright-exclude file](https://github.com/model-checking/kani/blob/main/scripts/ci/copyright-exclude).

File: papers/vstte2024/paper.tex [hunk 2]
```diff
@@ -0,0 +1,169 @@
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
```
Comment (celinval) [inline papers/vstte2024/paper.tex:2]: Comments in latex start with `%`

Comment (rahulku) [reply papers/vstte2024/paper.tex]: fixed

**Comment by rahulku** [comment]
removed LLNCS specific files, added README.

File: papers/vstte2024/splncs04.bst [hunk 3]
```diff
@@ -0,0 +1,1551 @@
...
+%% This was file `titto.bst',
+%% generated with the docstrip utility.
+%%
+%% The original source files were:
+%%
+%% merlin.mbs  (with options: `vonx,nm-rvvc,yr-par,jttl-rm,volp-com,jwdpg,jwdvol,numser,ser-vol,jnm-x,btit-rm,bt-rm,edparxc,bkedcap,au-col,in-col,fin-bare,pp,ed,abr,mth-bare,xedn,jabr,and-com,and-com-ed,xand,url,url-blk,em-x,nfss,')
+%% ----------------------------------------
+%% *** Tentative .bst file for Springer LNCS ***
+%%
+%% Copyright 1994-2007 Patrick W Daly
```
Comment (celinval) [inline papers/vstte2024/splncs04.bst:48]: Do we need this file? This also has a different `Copyright`.

Comment (rahulku) [reply papers/vstte2024/splncs04.bst]: we do not. i removed it. i missed that one. thanks for the check.

File: papers/vstte2024/README.md [hunk 4]
```diff
@@ -0,0 +1 @@
+This contains the contents for a short paper at VSTTE2024. In order to build this, please download the LLNCS style file available at https://resource-cms.springernature.com/springer-cms/rest/v1/content/19238648/data/v8 -o llncs.zip and use standard LaTeX tools to build the paper.
```
Comment (tautschnig) [inline papers/vstte2024/README.md:1]:
```suggestion
This contains the contents for a short paper at VSTTE2024. In order to build this, please download the LLNCS style file available at https://resource-cms.springernature.com/springer-cms/rest/v1/content/19238648/data/v8, unpack the resulting llncs.zip, and use standard LaTeX tools to build the paper. 
```

**Review by tautschnig — APPROVED** [review]

**Pull request merged**